src/components/Logo.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from "react"
+import { Avatar } from "./material/core"
+import config from "../utility/config"
+
+export interface LogoProps {
+  className?: string | undefined
+}
+
+const getLogoPath = () => `${process.env.PUBLIC_URL}/static/images/logo.png`
```

Comment:
You ok with this Boss @dipeshwalia ?

Reply:
this is ok for now, but its good to avoid any env on ui. 
Later on we can have configuration service to feed those value to Ui.

Reply:
I'm gonna add an issue for that